.github/get_issues.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+from datetime import date, datetime
+import csv
+import os
+
+import pandas as pd
+
+from storypoints import ZenHubAPI, GitHubAPI
+
+DEVTEAM_PROJECTS = [
+    'cdh-web', 'winthrop-django', 'django-pucas', 'djiffy', 'django-annotator-store',
+    'derrida-django', 'mep-django', 'viapy',
+    'ppa-django', 'parasolr', 'pemm-scripts', 'startwords'
+]
+
+def get_issues():
+    """
+    Use ZenHub and GitHub API to compile a CSV of all issues in the DEVTEAM_PROJECTS
+    list. If the CSV already exists, find the latest date in the CSV and update 
+    as necessary. Place the file into the `data` dir as issues.csv
+    """
+
+    # Get APIs
+    ghub = GitHubAPI(os.environ['GITHUB_USERNAME'], os.environ['GITHUB_API_TOKEN'])
+    zhub = ZenHubAPI(os.environ['ZENHUB_API_TOKEN'])
+
+    # Read existing CSV
+    filename = 'data/issues.csv'
+    if os.path.exists(filename):
+        df = pd.read_csv(filename, parse_dates=['closed'])
+        # find the latest date already read
+        date_since = df['closed'].max()
+        data_dict = df.to_dict('records')
+    else:
+        date_since = date(2020, 7, 1) # HELP: How far back do we want to look?
+        data_dict = []
+
+    # get dict of repo by name -> id
+    repos = ghub.get_org_repos('Princeton-CDH')
+
+    for repo, repo_id in repos.items():
+        if repo not in DEVTEAM_PROJECTS:
+            continue
+
+        for issue in ghub.get_repo_issues('Princeton-CDH', repo,
+                                          params={'since': date_since.isoformat()}):
+            
+            zhub_info = zhub.get_issue(repo_id, issue['number'])
+            
+            info = {
+                'issue': issue['title'],
+                'project': repo,
+                'url': issue['url'],
+                'estimate': zhub_info.get('estimate', {}).get('value', ''),
+                'closed': issue['closed_at'],
+                'labels': ';'.join([label['name'] for label in issue['labels']])
+            }
+
+            data_dict.append(info)
+
+    pd.DataFrame(data_dict).to_csv(filename, index=False)
```

Comment:
Correct haha, when you have a hammer everything looks like a nail. Though, I now realize that our "since" will probably create duplicate rows, which in Pandas is easy to fix. So in the next round of corrections I'll need pandas in this file